scripts/sct_label_vertebrae.py

```diff
@@ -315,8 +315,7 @@ def main(args=None):
             # which does not apply a convolution across all voxels (highly inneficient)
             im_label.save(fname_labelz)
         elif fname_initlabel:
-            import sct_label_utils
-            sct_label_utils.main(['-i', fname_initlabel, '-add', '-1', '-o', fname_labelz])
+            Image(fname_initlabel).save(fname_labelz)
```

Comment:
no need to read and save the same file: 
```suggestion
            fname_labelz = fname_initlabel
```